src/Monitor/Inqueues.vue

```diff
@@ -2,7 +2,12 @@
 <div>
   <toggleBar></toggleBar>
   <b-collapse v-model="showCollapse" id="collapseQueueManager" class="mt-2">
-    <b-table style="margin-top:10px" striped hover responsive :items="inqueues" :fields="fields">
+    <b-table style="margin-top:10px" striped hover small responsive
```

Comment:
You can unify it with btable by using formatter, I think. Instead of slot.

Reply:
you mean use btable instead of b-table component?

Reply:
Yeah, sort of. I remember I did it somewhere. Not sure it will work here, though.